Remote Habits/View/RHLoginViewController.swift

```diff
@@ -122,13 +122,19 @@ class RHLoginViewController: RHBaseViewController {
         loginButton.isEnabled = false
         loginButtonState()
         
-        userManager.isGuestLogin = isGuestLogin
         if let viewController  = UIStoryboard(name: RHConstants.kStoryboardMain, bundle: nil).instantiateViewController(withIdentifier: RHConstants.kDashboardViewController) as? RHDashboardViewController {
             viewController.isSourceLogin = true
             navigationController?.pushViewController(viewController, animated: true)
         }
     }
     
+    func fillHabitData() {
```

Comment:
```suggestion
    func populateInitialHabitData() {
```

I think a more descriptive function name would be good to use here. `fillHabitData` doesn't describe what the function is doing.

Reply:
Done.